lib/class-wp-rest-plugins-controller.php

```diff
@@ -41,11 +41,11 @@ public function register_routes() {
 	 * Check if a given request has access to read /plugins.
 	 *
 	 * @param  WP_REST_Request $request Full details about the request.
-	 * @return WP_Error|boolean
```

Comment:
We should actually be returning `WP_Error` on failure as much as possible, as it gives us more descriptive errors:

![image](https://cloud.githubusercontent.com/assets/36432/12681110/d5c78b54-c661-11e5-81c1-efa8aa4e3192.png)